_includes/4-action-page-questions.html

```diff
@@ -1,6 +1,11 @@
 <div class="small-12 medium-4 columns">
   <div>
+    {% if page.path contains 'job-seekers' %}
```

Comment:
Is this always going to be "Questions" with any page that does not contain `job-seekers`? Does is make more sense to pass this title from the markdown file like `{{sidebar-heading}}` so each page can have the title of their choosing?